eRecruiter.Api.Client/Requests/Applicant/ApplicantExternalIdGetRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using eRecruiter.Api.Responses;
+
+namespace eRecruiter.Api.Client.Requests
+{
+    public class ApplicantExternalIdGetRequest : HttpRequestMessage<ApplicantExternalIdResponse>
+    {
+        public ApplicantExternalIdGetRequest(int externalId)
+            : base(HttpMethod.Get, "Api/Applicant/ExternalId/" + externalId)
```

Comment:
We should handle this like the e-mail address and use ``Api/Applicants/ByExternalId/{email}`` to query for the external ID.